### PR TITLE
Update webhook user guide

### PR DESCRIPTION
**Pull request by mmpei**
Signed-off-by: peimingming <peimingming@corp.netease.com>

### CODE REVIEW

**Comment by mmpei** [comment]
@wy65701436 @xaleeks

File: docs/working-with-projects/project-configuration/configure-webhooks.md [hunk 1]
```diff
@@ -9,81 +9,120 @@ The action that is taken upon receiving a notification from a Harbor project dep
 
 ### Supported Events
 
-You can define one webhook endpoint per project. Webhook notifications provide information about events in JSON format and are delivered by `HTTP` or `HTTPS POST` to an existing webhhook endpoint URL that you provide. The following table describes the events that trigger notifications and the contents of each notification.
+You can define multiple webhook endpoints per project. Harbor supports two kinds of endpoints currently, One type is `HTTP`  and the other is `SLACK`. Webhook notifications provide information about events in JSON format and are delivered by `HTTP` or `HTTPS POST` to an existing webhhook endpoint URL or Slack address that you provide. The following table describes the events that trigger notifications and the contents of each notification.
```
Comment (xaleeks) [inline docs/working-with-projects/project-configuration/configure-webhooks.md:12]: please change to  -> ’Harbor supports two kinds of endpoints currently, `HTTP` and `SLACK`. ‘

File: docs/working-with-projects/project-configuration/configure-webhooks.md [hunk 2]
```diff
@@ -9,81 +9,120 @@ The action that is taken upon receiving a notification from a Harbor project dep
...
 
-    ![Webhook URL](../../../img/webhooks2.png)
+    ![Webhook URL](../../img/webhooks2.png)
 
 1. Click **Test Endpoint** to make sure that Harbor can connect to the listener.
+
 1. Click **Continue** to create the webhook.
 
-When you have created the webhook, you see the status of the different notifications and the timestamp of the last time each notification was triggered. You can click **Disable** to disable notifications. 
+When you have created the webhook, you see the list of webhooks. If you click the arrow at the left of the row, can you see the status of the different notifications and the timestamp of the last time each notification was triggered. You can also manage the webhook by clicking the drop list button of `ACTION...` .
```
Comment (xaleeks) [inline docs/working-with-projects/project-configuration/configure-webhooks.md:121]: please change to 

'When you have created the webhook, you can click on the arrow at the left end to see the status of the different notifications and the timestamp of the last time each notification was triggered. You can also manage the webhook by clicking the drop list button of `ACTION...` .'

File: docs/working-with-projects/project-configuration/configure-webhooks.md [hunk 3]
```diff
@@ -9,81 +9,120 @@ The action that is taken upon receiving a notification from a Harbor project dep
...
 
-{{< note >}}
-You can only disable and reenable all notifications. You cannot disable and enable selected notifications.
-{{< /note >}}
+You can modify the webhook, you can also `Enable` or `Disable` the webhook.
 
-![Webhook Status](../../../img/webhooks3.png)
+![Webhook Status](../../img/webhooks3.png)
```
Comment (xaleeks) [inline docs/working-with-projects/project-configuration/configure-webhooks.md:125]: '../../../' is correct, this is a behavior of the system that hooks up to the website, basically the img is in its own directory, this needs to be changed back or image won't show properly on website

File: docs/working-with-projects/project-configuration/configure-webhooks.md [hunk 4]
```diff
@@ -94,4 +133,4 @@ As a Harbor system administrator, you can enable and disable webhook notificatio
 1. Go to **Configuration** > **System Settings**.
 1. Scroll down and check or uncheck the **Webhooks enabled** check box.
 
-    ![Enable/disable webhooks](../../../img/webhooks4.png)
+    ![Enable/disable webhooks](../../img/webhooks4.png)
```
Comment (xaleeks) [inline docs/working-with-projects/project-configuration/configure-webhooks.md:136]: '../../../' is correct, this is a behavior of the system that hooks up to the website, basically the img is in its own directory, this needs to be changed back or image won't show properly on website

File: docs/working-with-projects/project-configuration/configure-webhooks.md [hunk 5]
```diff
@@ -9,81 +9,120 @@ The action that is taken upon receiving a notification from a Harbor project dep
...
+1. Select events that you want to subscribe.
 
 1. Enter the URL for your webhook endpoint listener.
+
 1. If your webhook listener implements authentication, enter the authentication header. 
+
 1. To implement `HTTPS POST` instead of `HTTP POST`, select the **Verifiy Remote Certficate** check box.
 
-    ![Webhook URL](../../../img/webhooks2.png)
+    ![Webhook URL](../../img/webhooks2.png)
```
Comment (xaleeks) [inline docs/working-with-projects/project-configuration/configure-webhooks.md:115]: '../../../' is correct, this is a behavior of the system that hooks up to the website, basically the img is in its own directory, this needs to be changed back or image won't show properly on website

File: docs/working-with-projects/project-configuration/configure-webhooks.md [hunk 6]
```diff
@@ -9,81 +9,120 @@ The action that is taken upon receiving a notification from a Harbor project dep
...
   - If the scan passes, deploy the image.
 
 ### Configure Webhooks
 
 1. Log in to the Harbor interface with an account that has at least project administrator privileges.
+
 1. Go to **Projects**, select a project, and select **Webhooks**.
 
-    ![Webhooks option](../../../img/webhooks1.png)
+    ![Webhooks option](../../img/webhooks1.png)
```
Comment (xaleeks) [inline docs/working-with-projects/project-configuration/configure-webhooks.md:103]: '../../../' is correct, this is a behavior of the system that hooks up to the website, basically the img is in its own directory, this needs to be changed back or image won't show properly on website

File: docs/working-with-projects/project-configuration/configure-webhooks.md [hunk 7]
```diff
@@ -9,79 +9,118 @@ The action that is taken upon receiving a notification from a Harbor project dep
...
 
 1. Log in to the Harbor interface with an account that has at least project administrator privileges.
+
 1. Go to **Projects**, select a project, and select **Webhooks**.
 
     ![Webhooks option](../../../img/webhooks1.png)
 
+1. Select notify type `HTTP`, so the webhook will be send to a HTTP endpoint.
+
+1. Select events that you want to subscribe.
```
Comment (xaleeks) [inline docs/working-with-projects/project-configuration/configure-webhooks.md:107]: 'that you want to subscribe to'

**Pull request merged**